_config/site.yml

```diff
@@ -98,6 +98,7 @@ profiles:
     js_minifier: disabled
     html_minifier: disabled
     legacy_post_syntax_highlighting: disabled
+    ignore_older_than_days: 30
```

Comment:
Since it's already done, I would keep it like this.

We can create an issue for this in the future if the need arise.